presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -102,8 +102,67 @@ Property Name                                      Description
 ``hive.immutable-partitions``                      Can new data be inserted into existing partitions?           ``false``
 
 ``hive.max-partitions-per-writers``                Maximum number of partitions per writer.                     100
+================================================== ============================================================ ==========
+
+Amazon S3 Configuration
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The Hive connector also allows querying data in Hive where data files are stored in Amazon S3. 
+The SQL support for S3 tables remains the same as for HDFS tables.
+
+For accessing tables stored in S3 the mandatory configuration properties required
+are the AWS credential properties ``hive.s3.aws-access-key`` and ``hive.s3.aws-secret-key``. 
+You can also use ``hive.s3.use-instance-credentials`` which if set to true, enables retrieving
```

Comment:
Or you could say --
"There are two ways to enable access to S3 data via hive connector
1) Set the values of AWS credential properties `hive.s3.aws-access-key` and `hive.s3.aws-secret-key`.
2) Set `hive.s3.use-instance-credentials` to true. This gives you temporary AWS credentials that have the same permissions as that of the IAM role associated with the EC2 instance in its instance profile....."